fh-android-sdk/src/main/java/com/feedhenry/sdk/FHDataManager.java

```diff
@@ -0,0 +1,167 @@
+package com.feedhenry.sdk;
+
+import android.content.Context;
+import com.feedhenry.sdk.datamanager.FHStore;
+import org.jboss.aerogear.android.store.DataManager;
+import org.jboss.aerogear.android.store.Store;
+import org.jboss.aerogear.android.store.memory.EncryptedMemoryStoreConfiguration;
+import org.jboss.aerogear.android.store.memory.MemoryStoreConfiguration;
+import org.jboss.aerogear.android.store.sql.EncryptedSQLStore;
+import org.jboss.aerogear.android.store.sql.EncryptedSQLStoreConfiguration;
+import org.jboss.aerogear.android.store.sql.SQLStoreConfiguration;
+
+import java.security.spec.InvalidKeySpecException;
+
+public final class FHDataManager {
+
+    private Context context;
+
+    private FHDataManager(Context context) {
+        this.context = context;
+    }
+
+    public static FHDataManager with(Context context) {
+        return new FHDataManager(context);
+    }
+
+    public FHStore getStore(String storeName) {
+        Store store = DataManager.getStore(storeName);
+        if (store == null) {
+            throw new IllegalStateException("Store '" + storeName + "' not found");
+        }
+        return wrapperStore(store);
+    }
+
+    public FHMemoryStoreInitializer memoryStore(String name) {
+        return new FHMemoryStoreInitializer(name);
+    }
+
+    public FHEncryptedMemoryStoreInitializer encryptedMemoryStore(String name) {
+        return new FHEncryptedMemoryStoreInitializer(name);
+    }
+
+    public FHSQLStoreInitializer sqlStore(String name) {
+        return new FHSQLStoreInitializer(name);
+    }
+
+    public FHEncryptedSQLStoreInitializer encryptedSqlStore(String name) {
+        return new FHEncryptedSQLStoreInitializer(name);
+    }
+
+    private FHStore wrapperStore(Store store) {
+        return new FHStore(store);
+    }
+
+    public abstract class FHStoreInitializer {
```

Comment:
Why not? keep in mind, it's only a wrapper under another library so, anyone will use it out of this. Anyway we are always open for suggestion, if you have any shoot it and we will discuss about it